tests/unit/script/size_of.rs

```diff
@@ -40,10 +40,10 @@ macro_rules! sizeof_checker (
 // Update the sizes here
 sizeof_checker!(size_event_target, EventTarget, 40);
 sizeof_checker!(size_node, Node, 160);
-sizeof_checker!(size_element, Element, 336);
-sizeof_checker!(size_htmlelement, HTMLElement, 352);
-sizeof_checker!(size_div, HTMLDivElement, 352);
-sizeof_checker!(size_span, HTMLSpanElement, 352);
+sizeof_checker!(size_element, Element, 328);
```

Comment:
Looks like I hadn’t pushed my last commit which reverts this. (It adds a couple `bool` flags but alignment rounds to 8 bytes) Sorry :)